libs/minizip/Makefile

```diff
@@ -0,0 +1,61 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
```

Comment:
https://github.com/openwrt/packages/blame/master/CONTRIBUTING.md#L30

Reply:
Well, these were mostly lifted from existing files © OpenWRT but I don't object to claiming ownership of them. Fixed.